.github/workflows/pr_check.yml

```
@@ -1,19 +1,34 @@
-name: pr check
+name: Pull Request Check
 on:
   pull_request:
     types: [ synchronize ]
 
 jobs:
   build:
-    runs-on: ubuntu-20.04
-    container:
-      image: openjdk:8-jdk
+    strategy:
+      matrix:
+        java: [
+            1.8, # Minimum
```

```suggestion
            8, # Minimum
```
Required for `actions/setup-java@v2`